locallibs/install.py

```diff
@@ -97,15 +97,15 @@ def install_extras(framework_path, version="2.7", requirements_file=None,
         print('*********************************************************')
         print()
     ensure_pip(framework_path, version)
-    if upgrade_pip:
-        upgrade_pip_install(framework_path, version)
-    if requirements_file:
-        install_requirements(requirements_file, framework_path, version)
-    elif version.startswith("2."):
+    if version.startswith("2."):
```

Comment:
Doesn't this mean you no longer have the option of not installing the extra packages? The previous structure resulted in the PYTHON[2,3]_EXTRA_PKG being installed if there was no requirements file. If there was a requirements file, only the things in that file were installed.

With your changes, you get the extra packages always. So I cannot build a framework that does not include PyObjC.

Am I missing something?

Reply:
Nope, you nailed it. Let me rethink the logic a bit more. I think I was a little aggressive and totally spaced the want/need for a package-less framework.

Reply:
Should be fixed in fe9f734. Ran through a test for vanilla, and with upgraded pip, built as expected. Vanilla. Running through our requirements to ensure same behavior as before (expected)

Reply:
And fresh build works with our requirements.